### PR TITLE
Turn controller scaffold

### PR DESCRIPTION
**Pull request by atd**

### CODE REVIEW

File: app/controllers/turns_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  end
+
+  # DELETE /turns/1
+  def destroy
+    @turn.destroy
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
+    def set_turn
```
Comment (nogates) [inline app/controllers/turns_controller.rb]: indentation >:P

**Review by nogates — APPROVED** [review]
:clap:

File: app/controllers/turns_controller.rb (hunk 1 above)
Comment (atd) [reply app/controllers/turns_controller.rb]: It is rails generator's fault!

File: api.raml [hunk 2]
```diff
@@ -4,233 +4,399 @@ version: v1
...
-            "name": "Pepe y Lola",
-            "emails": [
-              "pepe@mat.com",
-              "lola@mat.com"
-            ]
+            "data": {
+              "type": "groups",
+              "attributes": {
+                "name": "Pepe y Lola",
+                # FIXME Internally, these emails are mapped to users, with an email
```
Comment (atd) [inline api.raml:298]: @nogates I need feedback on this

**Pull request merged**